_drafts/SC3-1-1-html-has-lang.md

```diff
@@ -0,0 +1,42 @@
+---
+name: SC3-1-1-html-has-lang
+
+description: |
+  This rule checks that the `html` element has a `lang` or `xml:lang` attribute.
+
+success_criterion:
+- 3.1.1
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Annika Nietzio
+- Jey Nandakumar
+---
+
+## Test Procedure
+
+### Applicability
+
+The root element of the page, if it is an `html` element.
+
+### Expectation
+
+The test target has a `lang` or `xml:lang` attribute.
```

Comment:
Add a note:
> **Note**: HTML 5 recommends using `lang` instead of `xml:lang`. This is not known to impact accessibility, which is why use of both is permitted by this rule.